stroeer/core/v1/article.proto

```diff
@@ -8,133 +8,317 @@ option go_package = "github.com/stroeer/tapir/go/stroeer/core/v1;core";
 option java_multiple_files = true;
 option java_package = "de.stroeer.core.v1";
 
+// An article represents a piece of content created in the content management
+// system. Different types of content like text or video articles share
+// the same message structure distinguished by the `Article.type` field.
+//
+// Depending on the service used to retrieve an article, this message might
+// only contain data required on overview pages (e.g. with `Article.body` set to `null` thus not containing any data that is only required on detail pages). This is sometimes referred to as _Teaser_.
+//
+// Message fields and map entries are optional unless commented otherwise.
 message Article {
+  // Unique ID of the article defined by the content management system (required).
   int64 id = 1;
+
+  // Main content type of the article.
   ContentType type = 2;
+
+  // Subtype of the article. For `CONTENT_TYPE_ARTICLE` this field holds a sub type, for others like `CONTENT_TYPE_GALLERY` it may not.
   ContentSubType sub_type = 3;
+
+  // References to get access to the article on different technical platforms like
+  // canonical web or AMP (required).
   References references = 4;
+
+  // Hierarchical section information of the article (required).
   Section section = 5;
+
+  // Generic map containing general content and configuration information of
+  // the article (required).
+  //
+  // The entry set is defined by the content management system and
+  // will vary depending on the main type of the article.
+  //
+  // For CONTENT_TYPE_ARTICLE, this map will always contain non empty values
+  // for the following keys:
+  //
+  // * headline
+  // * top_line
+  // * summary
+  //
+  // For CONTENT_TYPE_GALLERY, this map will always contain non empty values
+  // for the following keys:
+  //
+  // * headline
+  //
+  // Clients must be resilient to unknown or missing entry sets.
   map<string, string> fields = 6;
+
+  // Recursive body structure of the article to render on detail
+  // pages.
+  //
+  // This message will be empty for the teaser representation of the article.
+  //
+  // see https://github.com/orgs/stroeer/teams/api/discussions/1
```

Comment:
Discussion is in a gh team only accessible to members of Team Buzz. 
I understand the motivation behind this comment, but would still remove it before merging.